target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_default "dsl" "DSL" "${board}:white:dsl" "0"
+	ucidef_set_led_usbport "usb" "USB" "${board}:white:usb" "usb4-port1" "usb2-port1"
+	ucidef_set_led_switch "lan" "lan" "${board}:white:lan" "switch0" "0x1e"
+	ucidef_set_led_default "wlan2g" "WLAN2g" "$board:white:wlan2g" "0"
+	ucidef_set_led_default "wlan5g" "WLAN5g" "$board:white:wlan5g" "0"
+	ucidef_set_led_switch "wan" "wan" "${board}:white:wan" "switch0" "0x20"
+	ucidef_set_led_default "general" "general" "${board}:white:ledgnr" "1"
```

Comment:
What about dropping this here and use the general led like the power led on other boards?

```
+	aliases {
+		serial0 = &uart4;
+		mdio-gpio0 = &mdio0;
+
+		led-boot = &ledgnr;
+		led-failsafe = &ledgnr;
+		led-running = &ledgnr;
+		led-upgrade = &ledgnr;
+	};
```